cli/tools/installer.rs

```diff
@@ -185,12 +189,15 @@ pub async fn infer_name_from_url(url: &Url) -> Option<String> {
 }
 
 pub fn uninstall(uninstall_flags: UninstallFlags) -> Result<(), AnyError> {
-  let name = uninstall_flags.name;
-  let root = uninstall_flags.root;
-
   if !uninstall_flags.global {
     log::warn!("⚠️ `deno install` behavior will change in Deno 2. To preserve the current behavior use the `-g` or `--global` flag.");
   }
+  let UninstallKind::Global(UninstallFlagsGlobal { name, root }) =
+    uninstall_flags.kind
+  else {
+    unreachable!();
```

Comment:
Not a big deal and feel free to ignore, but it might be slightly better to use a match statement here so that we get compiler errors on all the spots that need to change once implementing `UninstallKind::Local`. For example:

```rs
let uninstall_flags = match uninstall_flags.kind {
  UninstallKind::Global(flags) => flags,
  UninstallKind::Local => unreachable!(), // will do a compiler error in the future when this changes
};
```

Then we can be more sure to not miss something in the future.